runtime/compiler/rpc/raw/J9Client.cpp

```diff
@@ -242,6 +245,7 @@ J9ClientStream::J9ClientStream(TR::PersistentInfo *info)
    : J9Stream(),
    _timeout(info->getJITaaSTimeout())
    {
+   _numConnectionsOpened++;
```

Comment:
Let's move the increment at the end of the constructor, just in case the constructor fails.